src/main/java/uk/gov/ons/ctp/integration/rhsvc/representation/NewCaseDTO.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.ons.ctp.integration.rhsvc.representation;
+
+import java.time.LocalDate;
+import java.util.UUID;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.ons.ctp.common.log.LoggingScope;
+import uk.gov.ons.ctp.common.log.Scope;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class NewCaseDTO {
+
+  private int surveyId;
+  @NotNull private UUID collectionExerciseId;
+
+  @NotBlank private String schoolId;
+  @NotBlank private String schoolName;
+
+  private boolean consentGivenTest;
+  private boolean consentGivenSurvey;
+
+  @NotBlank private String firstName;
```

Comment:
It's not clear to me if more of these should be marked with 
```
@LoggingScope(scope = Scope.MASK)
```

Which ones are to be masked ? if they are all the ones in the "sampleSensitive" list then things like `firstName` and `surname` and several others need to be included